sites/all/modules/unl/unl.module

```diff
@@ -546,24 +546,11 @@ function unl_menu_alter(&$items) {
  * Custom page callback.
  */
 function unl_404_page() {
-  $database = $GLOBALS['databases']['default']['default'];
-  $database['prefix'] = unl_get_shared_db_prefix();
-  Database::addConnectionInfo('default_site', 'default', $database);
-  // Switch database connection to default site.
-  db_set_active('default_site');
-
-  $default_404 = unl_shared_variable_get('site_404');
-  // @TODO: $default_404 should look like "node/182" but is there a safer way to retrieve nid?
-  $node = node_load(substr($default_404, 5));
-  if ($node) {
-    drupal_set_title($node->title);
-    $build = node_view($node, 'full');
-    $build['links']['#access'] = FALSE;
-    $build['#contextual_links'] = NULL;
-  }
-  else {
-    $build = array();
-  }
+  drupal_add_css('sites/all/modules/unl/404/unl-404.css');
+  
+  $build = array(
+    '#markup' => file_get_contents(__DIR__ . '/404/404.html')
```

Comment:
I don't think that really matters as much, because `__DIR__` will always point to the correct path and `__DIR__` is probably much faster than `drupal_get_path`.  It matters a lot more with something like this `drupal_add_css('sites/all/modules/unl/404/unl-404.css');`